bag_verifier/src/main/scala/uk/ac/wellcome/platform/archive/bagverifier/services/NotificationService.scala

```diff
@@ -0,0 +1,101 @@
+package uk.ac.wellcome.platform.archive.bagverifier.services
+
+import grizzled.slf4j.Logging
+import uk.ac.wellcome.messaging.sns.{PublishAttempt, SNSWriter}
+import uk.ac.wellcome.platform.archive.bagverifier.models.BagVerification
+import uk.ac.wellcome.platform.archive.common.models.BagRequest
+import uk.ac.wellcome.platform.archive.common.progress.models.{Progress, ProgressNotice, ProgressUpdate}
+import uk.ac.wellcome.json.JsonUtil._
+
+import scala.concurrent.{ExecutionContext, Future}
+import scala.util.{Failure, Success, Try}
+
+class NotificationService(
+                           progressSnsWriter: SNSWriter,
+                           outgoingSnsWriter: SNSWriter
```

Comment:
Naming nit: both of these writers are outgoing; I think we use “ongoing” in some other places?

Reply:
Arguably both are ongoing too - we reference ongoing in terraform only, so i'll update that to outgoing so it's consistent.

Reply:
You've also used ongoing elsewhere in this patch!

I wonder if “pipeline” might be a better word?